syntax = "proto3";

import "google/protobuf/empty.proto";
option objc_class_prefix = "SPLAB";

package splab_grpc_service;

service SPLabRenderingServer{
    // rpc initServer(google.protobuf.Empty) returns (google.protobuf.Empty) {}
    // rpc SayHello (HelloRequest) returns (HelloReply){}
    rpc init_rendering_server(google.protobuf.Empty) returns (google.protobuf.Empty) {}
}
